sample/ACPAnalyticsSample/android/build.gradle

```diff
@@ -25,6 +25,7 @@ allprojects {
         mavenLocal()
         google()
         jcenter()
```

Comment:
@cacheung can we also remove jcenter() ? (ref https://blog.gradle.org/jcenter-shutdown)

Reply:
updated.